.expeditor/verify.pipeline.yml

```diff
@@ -1,35 +1,33 @@
 steps:
   - label: bats
     command: bats /ci-studio-common/tests
-    agents:
-      queue: 'docker-privileged'
     plugins:
-      docker#v1.1.1:
-        image: "buildkite/plugin-tester"
-        workdir: /ci-studio-common
+      - docker#v3.0.0:
+          image: "buildkite/plugin-tester"
```

Comment:
💭 since your at it, these `"` are  also superfluous, aren't they?

Reply:
technically :)